lib/sqlalchemy/testing/provision.py

```diff
@@ -98,214 +95,43 @@ def _configs_for_db_operation():
 
 
```

Comment:
**mike bayer** (zzzeek) wrote:

starts w/ an adverb but that's still a "verb-like" beginning, this is imperative so that's good !

Reply:
**Gord Thompson** (gordthompson) wrote:

Ack